stdlib/atomic.ml

```diff
@@ -2,7 +2,7 @@
 (*                                                                        *)
 (*                                 OCaml                                  *)
 (*                                                                        *)
-(*             Gabriel Scherer, projet Partout, INRIA Paris-Saclay        *)
+(*                         The OCaml programmers                          *)
```

Comment:
I would recommend that you not be shy and follow the usual format here (even though of course there is arguably little copyright¹ in a one-line module including another; it's still a way to mark your work).

¹: copyright which would have been held by your employer anyway...

Reply:
ok...